docs/pages/guides/using-bun.mdx

```diff
@@ -0,0 +1,98 @@
+
+```bash
+bun run ios
+```
+
+And install packages with `bun expo install`
+
+```bash
+bun expo install expo-av
+```
+
+## Using Bun for EAS builds
+
+EAS will decide which packager to use based the lockfile in your codebase. So if you want EAS to use Bun, run `bun install` in your codebase and ensure it creates a **bun.lockb** - the Bun lockfile. Make sure to delete any other lockfiles. As long as this lockfile is in your codebase, Bun will be used as the package manager for your builds.
+
+### Customizing your Bun version on EAS
+
+EAS will use `bun@1.0.2` by default. If you want or need to use a particular version of Bun, you can configure the exact version in each build in your **eas.json**.
+
+```json eas.json
+{
+  "build": {
+    "test": {
+      "bun": "1.0.0"
+      // other settings...
+    }
+    // other build profiles...
+  }
+}
+```
+
+## Migrate to using Bun from yarn, npm or pnpm
+
+It is currently not possible to import another package manager's lockfile into Bun (though this feature is bieng [worked on](https://github.com/oven-sh/bun/issues/1751)). Until this is done, there is an element of risk to switching over to Bun on an existing project: you will have to remove your existing lockfile and `bun install` based on your **package.json**.
+
+The purpose of a lockfile is to _lock_ down your dependency tree. If there are any packages in **package.json** whose version numbers start with a `^` or `~`, you are likely to end up with a different version of the package. `^` means you're opting into future minor and patch versions, `~` means you're opting into future patch versions only. A lot of the time, you will not encounter problems, as according to SemVer minor and patch version shoudn't include any breaking changes. Unfortuantely things can easily slip through and we should always expect breaking changes to be a possibility. Your lockfile locks fown the _specific_ version and you won't get any updates unless you explicitly opt in. By deleting your lockfile, you are losing that safety net and getting the latest versions available of all the packages as defined in your **package.json**.
+
+To migrate to using Bun (use at your own risk):
+
+```bash
+rm -rf node_modules
+rm yarn.lock pnpm-lock.yaml package-lock.json
+bun install
+```
+
+## Trusted Dependencies
+
+Unlike other package manager, Bun does not automatically execute lifecycle scripts from installed packages as these could be considered a security risk. In particular, if a package you are installing has a `postinstall` script that you'd like to allow running, you will have to explicitly state that by including that package in your [`trusted dependencies`](https://bun.sh/guides/install/trusted) array in your **package.json**.
+
+The trust does not trickle down, meaning if you install `packageA` which has a dependency on `packageB` and `packageB` has a `postinstall` script, you will need to add `packageB` in your `trustedDependencies`.
```

Comment:
```suggestion
For example, if you install `packageA`, which has a dependency on `packageB` and `packageB` has a `postinstall` script, you must add `packageB` in your `trustedDependencies`.
```

I'm not sure what you meant by the first sentence here. Also, I assume you are trying to create a scenario and showcase it as an example.

Reply:
I meant that if you add a library to `trustedDependencies`, it won't automatically make _that_ library's dependencies also trusted. You still have to be explicit even if the library you want to trust is a dependency of a dependency.
But if it wasn't clear for you, it probably won't be for others so happy to remove that phrase.

Reply:
The context you just described is great! I didn't have that when I was reading so I was making the assumption on my part. If you think we should explain that to support that, feel free to do so.